src/credentials/credentialsUtilities.ts

```diff
@@ -12,6 +12,10 @@ import { credentialHelpUrl } from '../shared/constants'
 import { Profile } from '../shared/credentials/credentialsFile'
 import { isCloud9 } from '../shared/extensionUtilities'
 import { CredentialsProviderId, asString } from './providers/credentialsProviderId'
+import { createTimedPromise, Timeout } from '../shared/utilities/timeoutUtils'
+import { showMessageWithCancel } from '../shared/utilities/messages'
+
+const RESOLVE_PROVIDER_DEFAULT_TIMEOUT = 300000 // 5 minutes
```

Comment:
symbol name can be shorter without any loss of clarity, avoid verbose names. 

```suggestion
const CREDENTIALS_TIMEOUT = 300000 // 5 minutes
```